server/server_test.go

```diff
@@ -234,7 +236,7 @@ func TestServerStart(t *testing.T) {
 			Auth:                         true,
 			Debug:                        true,
 			MaximumFeedbackMessageLength: 255,
-		}, nil)
+		}, nil, mockStorageDVO)
```

Comment:
why is the DVO storage needed here and not the OCP one?

Reply:
Dan told me (mby i misunderstood) to add it there to have creating server with dvo storage tested.